offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/offer/overview/OfferOverviewView.groovy

```diff
@@ -68,7 +68,8 @@ class OfferOverviewView extends FormLayout {
         setupListeners()
     }
 
-    private void initLayout() {
+    private FormLayout generateDefaultLayout() {
```

Comment:
I think this method should come after the initLayout() method, which calls this method. Just for "cleaness" (NIT)